src/Config.php

```diff
@@ -1438,6 +1438,18 @@ function formatGlobalLayout(layout) {
             echo "</td></tr>";
         }
 
+        echo "<tr class='tab_bg_1'><th colspan='4' class='center b'>" . _n('Notification', 'Notifications', Session::getPluralNumber()) . "</th></tr>";
+
+        echo "<tr class='tab_bg_2'>";
+        echo "<td><label for='dropdown_is_notif_enable_default$rand'>" . __('Enable notifications by default') . "</label></td><td>";
```

Comment:
Same preference will be used to:
1. preselect "notification: no" in ITIL objects actor configuration, with ability to derogate to it;
2. disable all notifications on all other objects, without ability to derogate to it.

IMHO, it should be explained a bit, and maybe the `default` term should be removed here.
```suggestion
        echo "<td><label for='dropdown_is_notif_enable_default$rand'>" . __('Enable notifications') . "</label>&nbsp;"
        Html::showToolTip(__('It will be possible to reactivate notifications individually on certain elements, such as tickets, problems and changes.'));
        echo "</td><td>";
```

Reply:
done

![image](https://github.com/glpi-project/glpi/assets/7335054/f1cddd6d-7e6a-4bb6-a665-53a5ace33d18)